src/Microsoft.Framework.PackageManager/Scripts/ScriptExecutor.cs

```diff
@@ -67,18 +71,44 @@ public bool Execute(Runtime.Project project, string scriptName, Func<string, str
                 }
                 else
                 {
-                    var scriptCandiate = scriptArguments[0] + ".sh";
-                    if (File.Exists(scriptCandiate))
+                    // Special-case a script name that, perhaps with added .sh, matches an existing file.
+                    var surroundWithQuotes = false;
+                    var scriptCandidate = scriptArguments[0];
+                    if (scriptCandidate.StartsWith("\"", StringComparison.Ordinal) &&
```

Comment:
That's possible in the project.json file. But `CommandGrammar` removes spaces between and before all arguments and ignores the (remote) possibility of an argument that contains more than a correctly-quoted string.